app/src/main/java/com/infomaniak/mail/ui/main/thread/calendar/AttendeesBottomSheetDialog.kt

```diff
@@ -48,9 +48,8 @@ class AttendeesBottomSheetDialog : BottomSheetDialogFragment() {
         root.title = context.getString(R.string.attendeesListTitle, attendees.count())
         attendeeRecyclerView.adapter = AttendeesAdapter(attendees)
 
-        val dividerItemDecorator = DividerItemDecorator(
-            InsetDrawable(UiUtils.dividerDrawable(context), 16.toPx(), 0, 16.toPx(), 0)
-        )
-        attendeeRecyclerView.addItemDecoration(dividerItemDecorator)
+        val margin = context.resources.getDimensionPixelSize(R.dimen.dividerHorizontalPadding).toPx()
```

Comment:
`context.resources.getDimensionPixelSize(R.dimen.dividerHorizontalPadding)` already return somthing in pixel. We don't need to convert this value in pixel again.